cypress/e2e/pages/storyPage/index.cy.js

```diff
@@ -1,23 +1,263 @@
+    service: 'igbo',
+    runforEnv: 'live',
+    tests: canonicalTests,
+  },
+  {
+    path: '/igbo/afirika-23252735',
+    service: 'igbo',
+    runforEnv: ['local', 'test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/indonesia/dunia-53413801',
+    service: 'indonesia',
+    runforEnv: 'live',
+    tests: canonicalTests,
+  },
+  {
+    path: '/kyrgyz/kyrgyzstan-52891593',
+    service: 'kyrgyz',
+    runforEnv: 'live',
+    tests: canonicalTests,
+  },
+  {
+    path: '/kyrgyz/23292889',
+    service: 'kyrgyz',
+    runforEnv: 'test',
+    tests: canonicalTests,
+  },
+  {
+    path: '/news/uk-56342465',
+    service: 'news',
+    runforEnv: 'live',
+    tests: canonicalTests,
+  },
+  {
+    path: '/news/technology-56294493',
+    service: 'news',
+    runforEnv: 'live',
+    tests: canonicalTests,
+  },
+  {
+    path: '/news/23393110',
+    service: 'news',
+    runforEnv: 'test',
+    tests: canonicalTests,
+  },
+  {
+    path: '/newsround/56331357',
+    service: 'newsround',
+    runforEnv: 'live',
+    tests: canonicalTests,
+  },
+  {
+    path: '/newsround/23212028',
+    service: 'newsround',
+    runforEnv: 'test',
+    tests: canonicalTests,
+  },
+  {
+    path: '/pashto/world-52873295',
+    service: 'pashto',
+    runforEnv: 'live',
+    tests: canonicalTests,
+  },
+  {
+    path: '/pashto/23289748',
+    service: 'pashto',
+    runforEnv: 'test',
+    tests: canonicalTests,
+  },
+  {
+    path: '/sinhala/world-51723376',
+    service: 'sinhala',
+    runforEnv: 'live',
+    tests: canonicalTests,
+  },
+  {
+    path: '/sinhala/23225618',
+    service: 'sinhala',
+    runforEnv: 'test',
+    tests: canonicalTests,
+  },
+  {
+    path: '/sport/rugby-union/56359986',
+    service: 'sport',
+    runforEnv: ['live', 'local'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/sport/golf/56318994',
+    service: 'sport',
+    runforEnv: ['live', 'local'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/sport/formula1/23355387',
+    service: 'sport',
+    runforEnv: 'test',
+    tests: canonicalTests,
+  },
+  {
+    path: '/sport/tennis/23372108',
+    service: 'sport',
+    runforEnv: 'test',
+    tests: canonicalTests,
+  },
+  {
+    path: '/yoruba/afrika-58539527',
+    service: 'yoruba',
+    runforEnv: 'live',
+    tests: canonicalTests,
+  },
+  {
+    path: '/yoruba/afrika-23252769',
+    service: 'yoruba',
+    runforEnv: 'test',
+    tests: canonicalTests,
+  },
+];
+
+const testSuites = Cypress.env('SMOKE') ? smokeTestSuites : nonSmokeTestSuites;
+
+const ampTestSuites = testSuites.map(testSuite => {
+  return {
+    ...testSuite,
+    path: `${testSuite.path}.amp`,
+    tests: [...ampTests],
+  };
+});
+
+const liteTestSuites = testSuites
+  .filter(
+    ({ service }) =>
+      service !== 'news' && service !== 'sport' && service !== 'newsround',
+  )
+  .map(testSuite => {
+    return {
+      ...testSuite,
+      path: `${testSuite.path}.lite`,
+      tests: [...canonicalTests],
```

Comment:
We can import the lite tests from the article page tests
```suggestion
      tests: [...canonicalTests, ...liteTests],
```

Reply:
The lite tests are here: https://github.com/bbc/simorgh/blob/latest/cypress/e2e/pages/articles/testsForLiteOnly.js, but you can just import them into the storyPage file

Reply:
Yeah I imported this method but got this error: 

<img width="796" alt="image" src="https://github.com/user-attachments/assets/e47361d1-82dc-43e8-ae41-9c675065b3f5" />

Reply:
How did you import the tests - the format is probably incorrect, which is why you're getting this error. If you push your changes, I can have a look.

Reply:
done!